From 6e0e37519d485a64b3a332f25e45f8cc13dd61e0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 25 Jul 2016 18:46:59 +0200
Subject: calendar-server: Add back missing return value

In commit 7e0e224e0, when moving from e_cal_recur_generate_instances()
to e_cal_client_generate_instances(), the return value of the
ECalRecurInstanceFn callback was accidentally removed; add it
back.

https://bugzilla.gnome.org/show_bug.cgi?id=769156
---
 src/calendar-server/gnome-shell-calendar-server.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/calendar-server/gnome-shell-calendar-server.c b/src/calendar-server/gnome-shell-calendar-server.c
index 5ff12f2..57bc71b 100644
--- a/src/calendar-server/gnome-shell-calendar-server.c
+++ b/src/calendar-server/gnome-shell-calendar-server.c
@@ -443,6 +443,8 @@ generate_instances_cb (ECalComponent *comp,
   occurrence->rid        = e_cal_component_get_recurid_as_string (comp);
 
   appointment->occurrences = g_slist_append (appointment->occurrences, occurrence);
+
+  return TRUE;
 }
 
 
-- 
cgit v0.12

